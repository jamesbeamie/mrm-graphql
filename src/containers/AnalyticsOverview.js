import React, { Fragment } from 'react';
import CheckinsAnalytics from '../../src/components/analytics/checkins';
import '../assets/styles/custom.scss';
import '../assets/styles/topmenu.scss';
import AverageMeetingListComponent from '../components/analytics/AverageMeetingList';
import ComposedBookedRooms from '../components/analytics/ComposedBookedRooms';
import AveragePieChartList from '../components/analytics/averagePieChart/AveragePieChartList';

const AnalyticsOverview = () => (
  <Fragment>
    <AveragePieChartList />
    <ComposedBookedRooms />
    <CheckinsAnalytics />
    <AverageMeetingListComponent />
  </Fragment>
);

export default AnalyticsOverview;
