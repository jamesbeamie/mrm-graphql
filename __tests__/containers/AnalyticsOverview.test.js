import React from 'react';
import { shallow } from 'enzyme';
import AnalyticsOverview from '../../src/containers/AnalyticsOverview';

describe('AnalyticsOverview Component', () => {
  const props = {
    queryCompleted: jest.fn(),
    dateValue: {
      startDate: '',
      endDate: '',
    },
  };
  const wrapper = shallow(<AnalyticsOverview {...props} />);
  it('renders properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
