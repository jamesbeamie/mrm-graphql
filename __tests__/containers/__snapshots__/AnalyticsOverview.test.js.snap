// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnalyticsOverview Component renders properly 1`] = `
<Fragment>
  <AveragePieChartList />
  <ComposedBookedRooms />
  <Checkins />
  <AverageMeetingList />
</Fragment>
`;
